### PR TITLE
Fix regression with translations in case of multiple nodes.

### PR DESCRIPTION
**Pull request by fschulze**

### CODE REVIEW

File: src/chameleon/compiler.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def __call__(self, name, engine):
                 target = template(
                     "translate(msgid, mapping=mapping, domain=__i18n_domain, context=__i18n_context, target_language=target_language)",
                     msgid=ast.Str(s=formatting_string),
-                    target_language=target_language,
+                    target_language="target_language",
```
Comment (malthe) [inline src/chameleon/compiler.py]: I think this should be:
```diff
-                    target_language="target_language",
+                    target_language=load("target_language"),
```

Comment (fschulze) [reply src/chameleon/compiler.py]: Both work, but I guess then it should also be used in the ``if len(nodes) == 1:`` case a few lines above.

**Comment by fschulze** [comment]
@malthe It would be fantastic to get a release with this. If you give me permissions here and on pypi, I can handle that if you want.

**Comment by malthe** [comment]
https://pypi.org/project/Chameleon/3.4/ – out now.

**Comment by fschulze** [comment]
Thanks!

**Pull request merged**